client/src/Movies/SavedList.js

```diff
@@ -1,19 +1,25 @@
-import React, { Component } from 'react';
-
+import React, { Component } from 'react'
+import { NavLink, Link } from 'react-router-dom';
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
+  constructor(props){
+    super(props)
   }
 
-  render() {
+  render(){
     return (
       <div className="saved-list">
-        <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
-        ))}
-        <div className="home-button">Home</div>
+        <h3 >Saved Movies:</h3>
+        {this.props.list.map( movie => {
+          return (
+            <NavLink to={`/movies/${movie.id}`} key={movie.id} activeClassName="saved-active">
```

Comment:
good job adding these <NavLink /> components as a stretch goal. now this link will automatically render with an "active" className when the `to={ }` filed matches `this.props.params` >> now you can style your active links in the css file!